sycl/test-e2e/syclcompat/memory/memory_management_test3.cpp

```diff
@@ -31,6 +31,9 @@
 // ===----------------------------------------------------------------------===//
 // RUN: %{build} -o %t.out
 // RUN: %{run} %t.out
+//
+// UNSUPPORTED: linux && run-mode && level_zero_v2_adapter
```

Comment:
I think `level_zero_v2_adapter` is/should be a run-mode feature already, so `run-mode` could probably be dropped. Minor and not updates required though.

Reply:
ill just fix it, thanks